docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
       - bitcoind
       - bitcoind-signer
       - lnd1
+      - lnd2
```

Comment:
is that necessary? I know it outputs some error because lnd2 is not here, but assuming this is the basis or also the yaml for the "quickstart", it would be good to keep it slim.

maybe I could do a PR to make the yml/env more easily configurable, I think that could help.